src/internal/pfssync/cache_client.go

```diff
@@ -35,6 +35,27 @@ func NewCacheClient(pachClient *client.APIClient, renewer *renew.StringSet) *Cac
 	return cc
 }
 
+func (cc *CacheClient) GetFileset(commit *pfs.Commit) (string, error) {
```

Comment:
Should we be using a commit picker for this?

Reply:
I think that would be ideal for anything client side, at least in new code.  This is the internal client and "only for us", though, so we have some latitude on how friendly we want the API to be.  Since the new API just takes the ID returned from this, it's not a lot of extra work to resolve the commit yourself if it's important (there should probably be an API call for that; picker -> commit ID).

Reply:
It doesn't need one for the current usage since the client will always be working with resolved commits, but I think it would generally be better for us to always use resolved commits internally anyways.